CHANGES

```diff
@@ -17,3 +17,9 @@ Nothing
 - Add equality operator `==` and `eql?` method
 - Add `hash` method
 - Make id number instances immutable
+
+## [2.1.0] - 2023-04-05
+
+### Added
+
+- Support for ID numbers where the citizen is a refugee. (See https://www.oecd.org/tax/automatic-exchange/crs-implementation-and-assistance/tax-identification-numbers/South-Africa-TIN.pdf)
```

Comment:
Please set the version for this release to 2.2.0 as it's introducing a new feature. Also, please add it above the previous version.

Please also link to the doc with markdown, e.g.

```
Support for [ID numbers where the citizen is a refugee.](https://www.oecd.org/tax/automatic-exchange/crs-implementation-and-assistance/tax-identification-numbers/South-Africa-TIN.pdf)
```